src/Regions.tsx

```diff
@@ -29,7 +29,7 @@ const Regions = ({
 	names?: string | string[]
 	tags?: string | Record<string, string>
 	classes?: boolean | string | Record<string, string>
-}): React.JSX.Element[] => {
```

Comment:
Returning an Array should work from React 16.

https://stackoverflow.com/questions/39845443/react-typescript-return-array-of-jsx-elements